server.go

```diff
@@ -829,7 +829,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	if persistedNode != nil {
 		// If the color is still set to the default, we'll use the
 		// persisted color.
-		if cfg.Color == "#3399FF" {
+		if cfg.Color == defaultColor {
```

Comment:
I think you can create a separate commit where you export `defaultColor`. Then use it in your previous commit.  Then maybe delete this commit. `git rebase` can help.

Reply:
Yes, I agree. I'll squash it into one commit.